### ISSUE TITLE
remove dead instances from list of known instances

### ISSUE THREAD

**Issue by bsammon** [LemmyNet/lemmyBB#100]
### Steps to reproduce

The two known instances listed in the README.md appear to be down.

Maybe it's just a temporary situation.

### Version

main

**Comment by Tealk** [comment]
this currently makes no sense because the instances are only dead because the software has a bug:
https://github.com/LemmyNet/lemmyBB/issues/99

**Comment by Tealk** [comment]
Currently there are no working instances, so that might be some information to add.

**Comment by DraconicNEO** [comment]
> Currently there are no working instances, so that might be some information to add.

Yeah even the Lemmy-UI domain for the second one isn't working so it seems they've folded up.

**Comment by Tealk** [comment]
> it seems they've folded up

Oh no, we just changed the domain to roleplay.forum

**Comment by RokeJulianLockhart** [comment]
> https://github.com/LemmyNet/lemmyBB/issues/100#issue-1821544361

This persists. [^1] When I first saw them listed and noticed they were 404, I expected this project was unmaintained.

[^1]: [`blob/48bbf00b36cb79db78befa3bd5c59d1143bb8da7/README.md?plain=1#L23-L24`](https://github.com/LemmyNet/lemmyBB/blob/48bbf00b36cb79db78befa3bd5c59d1143bb8da7/README.md?plain=1#L23-L24:~:text=%7C-,Domain,-%7C%20Registration%20%7C)

**Comment by bsammon** [comment]
I concluded some time last year that this project was unmaintained.  Have you seen any activity that I've missed?

**Comment by RokeJulianLockhart** [comment]
> https://github.com/LemmyNet/lemmyBB/issues/100#issuecomment-2666925290

@bsammon, were it functional, ≈ 2 years of no commits wouldn't per se be the end of the world. God knows that SDDM sometimes gives projects like this a run for their money. However, you're probably right in this case.

**Comment by Nutomic** [comment]
The project is unmaintained. I added a note to the readme and archived the repo.
